arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -810,26 +719,20 @@ void RocksDBEngine::start() {
 
 #endif
 
-  _options.env->SetBackgroundThreads(static_cast<int>(opts._numThreadsHigh),
-                                     rocksdb::Env::Priority::HIGH);
-  _options.env->SetBackgroundThreads(static_cast<int>(opts._numThreadsLow),
-                                     rocksdb::Env::Priority::LOW);
+  _options.env->SetBackgroundThreads(
+      static_cast<int>(_optionsProvider.numThreadsHigh()),
+      rocksdb::Env::Priority::HIGH);
+  _options.env->SetBackgroundThreads(
+      static_cast<int>(_optionsProvider.numThreadsLow()),
+      rocksdb::Env::Priority::LOW);
 
-  // intentionally set the RocksDB logger to warning because it will
-  // log lots of things otherwise
   if (_debugLogging) {
     _options.info_log_level = rocksdb::InfoLogLevel::DEBUG_LEVEL;
-  } else {
-    if (!opts._useFileLogging) {
-      // if we don't use file logging but log into ArangoDB's logfile,
-      // we only want real errors
-      _options.info_log_level = rocksdb::InfoLogLevel::ERROR_LEVEL;
```

Comment:
We are not explicitly setting the log level to `ERROR_LEVEL` for normal operations.
Can this have a negative impact on performance, thanks to the way that log messages are evaluated by RocksDB? (just a speculation, haven't checked)

Reply:
This logic has been moved here:
https://github.com/arangodb/arangodb/blob/d6f0bf93597be53b27db6de6d8252016ee52a5d1/arangod/RocksDBEngine/RocksDBOptionFeature.cpp#L802-L806